### ISSUE TITLE
Captcha introduced by prolific. Script no more useful

### ISSUE THREAD

**Issue by Arslan30** [UnMars/Prolific-Joiner#40]
Since prolific has now introduced captcha with username and password, program has no way to handle it and is no more useful.

**Comment by oliveripkanam** [comment]
is it saying login failed for you as well?

**Comment by Arslan30** [comment]
yes. Upon analyzation of code, I think the problem is changes in the login page

**Comment by SurfingTurtles** [comment]
Hope someone can fix!

**Comment by Aji5ai** [comment]
Has anyone solved the captcha problem?

**Comment by Zero3K** [comment]
This script should stay broken. Scripts like this one are why common people aren't able to do studies as much as they used to be able to do.

**Comment by anberlin** [comment]
Had to end up switching to Bolts instead of this one due to issues with logging in.
